plugins/keyboard-navigation/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "blockly-plugin-keyboard-navigation",
+  "version": "0.20201029.0",
+  "description": "A Blockly plugin.",
+  "scripts": {
+    "audit:fix": "blockly-scripts auditFix",
+    "build": "blockly-scripts build",
+    "clean": "blockly-scripts clean",
+    "lint": "blockly-scripts lint",
+    "prepublishOnly": "npm run clean && npm run build",
+    "start": "blockly-scripts start",
+    "test": "blockly-scripts test"
+  },
+  "main": "./dist/index.js",
+  "module": "./src/index.js",
+  "unpkg": "./dist/index.js",
+  "author": "Blockly Team",
+  "private": true,
+  "keywords": [
+    "blockly",
+    "blockly-plugin",
+    "keyboard-navigation"
+  ],
+  "homepage": "https://github.com/alschmiedt/blockly-samples/tree/master/plugins/keyboard-navigation#readme",
+  "bugs": {
+    "url": "https://github.com/alschmiedt/blockly-samples/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/alschmiedt/blockly-samples.git",
+    "directory": "plugins/keyboard-navigation"
+  },
+  "license": "Apache 2.0",
```

Comment:
This will have to be `Apache-2.0` in order to parse correctly. I swear I fixed this too, so now I'm wondering if you're somehow using an older version of the script or if it didn't get updated correctly?

Reply:
I think I copied over the entire package.json from my branch which is pretty old. I updated so should have these changes.